src/newSeed/stage3.ts

```diff
@@ -75,13 +75,15 @@ export class Stage3 extends BaseStage {
     } else {
       // Check the token distribution
       let totalDistribAmount = BigNumber.from("0");
-      this.seedConfig.tokenDetails.tokenDistrib.forEach((tokenDistrb: { category: string, amount: string, lockup: number }) => {
-        if (!tokenDistrb.category) {
-          message = "Please enter a value for Category";
-        } else if (!tokenDistrb.amount || tokenDistrb.amount === "0") {
-          message = `Please enter a number greater than zero for Category ${tokenDistrb.category} Amount`;
-        } else if (!(tokenDistrb.lockup > 0)) {
-          message = `Please enter a number greater than zero for Category ${tokenDistrb.category} Lock-up`;
+      this.seedConfig.tokenDetails.tokenDistrib.forEach((tokenDistrb: { stakeHolder: string, amount: string, cliff: number, vest: number }) => {
+        if (!tokenDistrb.stakeHolder) {
+          message = "Please enter a value for Stakeholder";
```

Comment:
My final request was to make it  "${tokenDistrb.stakeHolder}".  Thing is, we also have to identify the row it is in, yes?

Reply:
the lines below say "Category", should also be "${tokenDistrb.stakeHolder}"

Reply:
Should we use the position like StakeHolder 1, Stakeholder 2 based on the position in the array ?

Reply:
What does it look like when you enter 30 chars?